prov/efa/src/rdm/rxr_ep.h

```diff
@@ -279,6 +285,57 @@ static inline int rxr_need_sas_ordering(struct rxr_ep *ep)
 	return ep->msg_order & FI_ORDER_SAS;
 }
 
+static inline uint64_t rxr_get_host_id(char *host_id_file)
+{
+	FILE *fp = NULL;
+	char *host_id_str = NULL;
+	const size_t HOST_ID_LENGTH = 16;
```

Comment:
should be 17 to include ending `\0`.

Reply:
Actually 16 also works. In this case null char is dropped silently at the end. Also it does not really matter for this function whether the string is null terminated, since we specified read length below.

Reply:
but `strtoul` require the input to be `\0` terminated string.

Reply:
Thanks. I realized that this is a bug. Fixed now.